client-shell/README.md

```diff
@@ -0,0 +1,108 @@
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# Palisade Shell
+
```

Comment:
I think that there is a need to let the reader know that they have to come "prepared".  We should not need to tell them how to get this example working, but at the same time tell them that they should come with a working knowledge of Kubernetes and will have already set-up the test environment as it is described in the Palisade-examples.  Could consider something short and sweet like: To work with with with this example, the test data will already need to be installed and working.  You should also confirmed it is setup running in a Kubernetes environment as described in the ?? section of the Palisade-examples....  In this way we make it clear that the reader needs to do some work first before doing anything here.

Reply:
I think the same applies to all the clients and is a job for the example to describe.
It doesn't necessarily have to be k8s, it could be the jvm deployment.

I will add a note referencing the example though